src/chrome/index.js

```diff
@@ -52,7 +52,7 @@ function noopChrome() {
       getURL(url) {
         return url;
       },
-      lastError: {message: ''},
+      lastError: undefined,
```

Comment:
Can we have a brief explanation on what this is about? Is it the most recent error message reported?

Reply:
I'm adding a comment at this exact spot. Additionally, this member is typed with the custom jsdoc typing at the moment and provides a link to the relevant entry in chrome extension api documentation.

```js
/**
 * @typedef Chrome.RuntimeError
 * @property {string} [message]
 * @see https://developer.chrome.com/docs/extensions/reference/runtime/#property-lastError
 */
```

Reply:
The state machine added in this changeset, checks lastError and reverts the state so it can try the next appropriate call. We can improve that further to handle more specific errors and forward unexpected ones to the panel to show the user. I set lastError in this object to undefined in this change as it shouldn't be set unless an error did occur.